### DISCUSSION TITLE
schedule restore

### DISCUSSION THREAD

**Discussion by ju187** [vmware-tanzu/velero#7912] [Community support Q&A]
Hi:

I am working on replicas some resources to the standby cluster.  They are backed up periodically with Velero on the active cluster.  Is there a way to set a schedule to restore the backup to another cluster?  If this is not supported, any other tool you could recommend?  Thank you

**Comment by sseago** [comment]
Velero doesn't have a restore schedule resource, but you might be able to use a CronJob resource to do it.

**Reply by kaovilai** [reply]
https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/
